go/kbfs/libkbfs/bserver_remote.go

```diff
@@ -612,7 +627,9 @@ func (b *BlockServerRemote) IsUnflushed(
 
 // GetUserQuotaInfo implements the BlockServer interface for BlockServerRemote
 func (b *BlockServerRemote) GetUserQuotaInfo(ctx context.Context) (info *kbfsblock.QuotaInfo, err error) {
-	ctx = rpc.WithFireNow(ctx)
+	if b.config.Mode().InitialDelayForBackgroundWork() == 0 {
```

Comment:
You're using this as a proxy for "am I on mobile"?   Can you add a comment about that?  I might have to change some stuff about this RPC anyway soon given HOTPOT-2091.

Reply:
Sort of, but I was  thinking if another mode has delayed background work it probably makes sense to not have FireNow here too.

Reply:
But why just this RPC, and none of the other ones?  Some OSes depends on seeing the quota in order to render the drive (e.g., space available) and so they get all the info at mount time.

Reply:
Because only this happens at init without any user input. I don't think this is from the OS -- I had mount-type=none when testing this.